src/components/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -21,6 +23,21 @@ const WorkspaceDetailsBreadcrumbs = (workspaceName: string) => [
   { name: 'Workspace Details', path: workspaceName ? `/workspaces/${workspaceName}` : '' },
 ];
 
+const WorkspaceActionMenu = (setDeleteModalOpen: (isOpen: boolean) => void) => ({
+  actions: [
+    {
+      id: '2',
+      label: 'Delete workspace',
+      cta: {
+        callback: () => {
+          setDeleteModalOpen(true);
+        },
+      },
+    },
+  ],
+  isDisabled: false,
```

Comment:
Maybe we can add a very future TODO here to check whatever roles or RBACs to check to see if you can delete said workspace